### PR TITLE
Acrolinx Batch 5 Edits

### PR DESCRIPTION
**Pull request by daphnemamsft**

### CODE REVIEW

File: docs-ref-conceptual/includes/troubleshoot-wsl.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ there's a good chance it's an issue with WSL and not with the CLI install proces
 To troubleshoot your WSL installation and possibly resolve issues:
 
 * If you can, run an identical install process on a Linux machine or VM to see if it succeeds. If it does,
-  your issue is almost certainly related to WSL. To start a Linux VM in Azure, see the
-  [create a Linux VM in the Azure Portal](/azure/virtual-machines/linux/quick-create-portal) documentation.
+  your issue is almost related to WSL. To start a Linux VM in Azure, see the
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/includes/troubleshoot-wsl.md:18]:
```suggestion
  your issue is likely related to WSL. To start a Linux VM in Azure, see the
```

File: docs-ref-conceptual/msal-based-azure-cli.md [hunk 2]
```diff
@@ -14,23 +14,23 @@ keywords: msal, msal-based azure cli
...
+Starting in version 2.30.0, Azure CLI uses [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-python) as the underlying authentication library. MSAL uses Azure Active Directory v2.0 authentication flow to provide more functionality and increases security for token cache.
 
 > [!WARNING]
 > BREAKING CHANGES are introduced in Azure CLI 2.30.0. Carefully read document prior to installation.
 
 ## `accessTokens.json` deprecation
 
-Previous versions of Azure CLI save ADAL tokens and service principal entries to `~/.azure/accessToken.json`. Latest versions of Azure CLI use MSAL and no longer generate `accessTokens.json`. Any existing workflow depending on `accessTokens.json` no longer works.
+Previous versions of Azure CLI save Microsoft Authentication Library tokens and service principal entries to `~/.azure/accessToken.json`. Latest versions of Azure CLI use MSAL and no longer generate `accessTokens.json`. Any existing workflow depending on `accessTokens.json` no longer works.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/msal-based-azure-cli.md:24]:
```suggestion
Previous versions of Azure CLI saved Azure AD Authentication Library (ADAL) tokens and service principal entries to `~/.azure/accessToken.json`. The latest versions of the Azure CLI use MSAL and no longer generate `accessTokens.json`. Any existing workflow depending on `accessTokens.json` no longer works.
```

File: docs-ref-conceptual/msal-based-azure-cli.md [hunk 3]
```diff
@@ -14,23 +14,23 @@ keywords: msal, msal-based azure cli
 
 # MSAL-based Azure CLI
 
-Starting in version 2.30.0, Azure CLI uses [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-python) as the underlying authentication library. MSAL uses AAD v2.0 authentication flow to provide more functionality and increases security for token cache.
+Starting in version 2.30.0, Azure CLI uses [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-python) as the underlying authentication library. MSAL uses Azure Active Directory v2.0 authentication flow to provide more functionality and increases security for token cache.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/msal-based-azure-cli.md:17]:
```suggestion
Starting in version 2.30.0, Azure CLI uses Microsoft Authentication Library [(MSAL)](https://github.com/AzureAD/microsoft-authentication-library-for-python) as the underlying authentication library. MSAL uses Azure Active Directory v2.0 authentication flow to provide more functionality and increases security for token cache.
```

File: docs-ref-conceptual/msal-based-azure-cli.md [hunk 4]
```diff
@@ -14,23 +14,23 @@ keywords: msal, msal-based azure cli
...
+The MSAL token cache and service principal entries are saved as encrypted files on Windows, and plaintext files on Linux and macOS.
 
 > [!IMPORTANT]
-> When using Azure CLI in a pipeline like Azure DevOps, ensure all tasks and stages are using versions of Azure CLI above v2.30.0 for MSAL-based Azure CLI. Azure CLI 2.30.0 is not backwards compatible with prior versions and throws an error when working with versions below 2.30.0.
+> When using Azure CLI in a pipeline like Azure DevOps, ensure all tasks and stages are using versions of Azure CLI above v2.30.0 for MSAL-based Azure CLI. Azure CLI 2.30.0 is not backwards compatible with prior versions and throws an error when working with versions prior to 2.30.0.
 
 ## Alternatives to consider
 
-Below are a couple alternatives you may consider for stability:
+A couple alternatives you may consider for stability:
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/msal-based-azure-cli.md:33]:
```suggestion
Alternatives you may consider for stability:
```

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]

File: docs-ref-conceptual/includes/cli-install-linux-dnf.md [hunk 5]
```diff
@@ -2,18 +2,18 @@
...
 ---
 
 ## Before you begin
 
-- Azure CLI [2.38.0](/cli/azure/release-notes-azure-cli#july-05-2022) (released on 07/05/2022) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft will not provide additional updates or bug fixes for these distributions.
+- Azure CLI [2.38.0](/cli/azure/release-notes-azure-cli#july-05-2022) (released on July 5, 2022) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft isn't providing other updates or bug fixes for these distributions.
 
-- Azure CLI and the Linux [Install script](/cli/azure/install-azure-cli-linux?pivots=script) will continue to be developed and supported by Microsoft. Packages provided by Red Hat and CentOS repositories are maintained and supported by their respective distributions.
+- Microsoft continues to develop and support Azure CLI and the Linux [Install script](/cli/azure/install-azure-cli-linux?pivots=script). Red Hat and CentOS distributions provide packages, while maintaining and supporting their respective repositories.
```
Comment (jiasli) [inline docs-ref-conceptual/includes/cli-install-linux-dnf.md:14]: > Red Hat and CentOS distributions provide packages, while maintaining and supporting their respective repositories.

This sentence is hard to understand to me.

Comment (dbradish-microsoft) [reply docs-ref-conceptual/includes/cli-install-linux-dnf.md:14]: @jiasli, This sentence had to be reworded to meet Acrolinx standards.

**Pull request merged**